components/common/Card.tsx

```diff
@@ -146,8 +156,9 @@ const Card = (props: CardProps) => {
 
   return (
     <StyledCard
-      className={`${negative && 'negative'} ${outline && 'outline'}`}
-      customcolor={customColor}
+      className={`${outline && 'outline'}`}
```

Comment:
Nitpick: This will apply `class="false"` to the DOM element

```suggestion
      className={outline && 'outline'}
```